packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.test.tsx

```diff
@@ -24,10 +22,6 @@ function renderWith(props?: ThirdPartyAuthProps) {
 }
 
 describe('ThirdPartyAuth component', () => {
-  let bundle: FluentBundle;
-  beforeAll(async () => {
-    bundle = await getFtlBundle('settings');
-  });
```

Comment:
We might could run `testAllL10n` here instead of removing it, or we can revisit whenever we get the l10n stuff more squared away.